sampleconfig/core.yaml

```diff
@@ -204,6 +204,18 @@ peer:
             # transaction's private data from other peers need to be skipped during the commit time and pulled
             # only through reconciler.
             skipPullingInvalidTransactionsDuringCommit: false
+            # implicitCollectionDisseminationPolicy specifies the dessemination policy for the peer's own implicit collection.
+            # When a peer endorses a proposal that writes to its own implicit collection, below values override the default values
+            # for disseminating private data.
+            # Note that it is applicable to all channels the peer has joined. The implication is that requiredPeerCount has to
+            # be smaller than the number of peers in a channel that has the lowest numbers of peers from the organization.
+            implicitCollectionDisseminationPolicy:
```

Comment:
I understand the scope of this PR (i.e., allow the user to configure the dissemination plan). The following is out of scope for this PR but would like to convey my point. I might be good to expose `memberOnlyRead` and `memberOnlyWrite` as the config parameter for implicit collection. Though the implicit collection is used by the chaincode lifecycle, there are ACL policies in place to restrict who can approve the chaincode definition, etc... However, for the user chaincode, the implicit collection allows any member to read and write as the default value of `memberOnlyRead` and `memberOnlyWrite` is false. Isn't it? On the other hand, as the collection configuration specified in the core.yaml is applicable to all channels, it may not make sense to expose these configs in core.yaml. Hence, it is better to leave the ACL policy to the chaincode. We might need to document this behavior (better to open a JIRA for this so that we don't miss).

Reply:
@cendhu Thanks... discussed with Manish and Jason and we decided memberOnlyRead and memberOnlyWrite is not strategic, we'll document in FAB-17373 that users should utilize chaincode logic instead.

Reply:
Yeah, I agree. If we make it a peer config, It would be applicable for all chaincodes which is not desirable.